cmd/serve.go

```diff
@@ -69,18 +79,42 @@ func cmdServe(cmd *cobra.Command, args []string) {
 		case <-ctx.Done():
 			return
 		case e := <-events:
-			switch e {
+			switch e.Type {
 			case kube.NewPodEvent:
-				metrics.GlobalPodMetrics.TotalNumberPods.Add(1)
+				podlog := log.New("pod", e.PodSelfLink)
+
+				cgPath := buildCgroupPath(cgroupRoot, e.PodUID, e.PodQOSClass)
```

Comment:
If k8s does not provide it, you might have to ask Docker the leader pid of the container and then check in `/proc/$pid/cgroup`...

You could file an issue and add a `// FIXME: ...` if you want to do that later.